examples/inverse/plot_evoked_ers_source_power.py

```diff
@@ -0,0 +1,144 @@
+# Reading the raw data and creating epochs:
+data_path = somato.data_path()
+subject = '01'
+task = 'somato'
+raw_fname = op.join(data_path, 'sub-{}'.format(subject), 'meg',
+                    'sub-{}_task-{}_meg.fif'.format(subject, task))
+
+raw = mne.io.read_raw_fif(raw_fname)
+
+# We are interested in the beta band. So we can filter
+raw.load_data().filter(12, 30)
+
+# Set picks, use a single sensor type
+picks = mne.pick_types(raw.info, meg='grad', exclude='bads')
+
+# Read epochs
+events = mne.find_events(raw)
+epochs = mne.Epochs(raw, events, event_id=1, tmin=-1.5, tmax=2, picks=picks,
+                    preload=True)
+
+# Read forward operator and point to freesurfer subject directory
+fname_fwd = op.join(data_path, 'derivatives', 'sub-{}'.format(subject),
+                    'sub-{}_task-{}-fwd.fif'.format(subject, task))
+subjects_dir = op.join(data_path, 'derivatives', 'freesurfer', 'subjects')
+
+fwd = mne.read_forward_solution(fname_fwd)
+
+# fix the subject in fwd['src']
+for src in fwd['src']:
+    src['subject_his_id'] = subject
+
+###############################################################################
+# now we can compute some source estimates using different methods.
+#
+# define the active and baseline windows in seconds
+# ERS activity starts at 0.5 seconds after stimulus onset
+active_win = (0.5, 1.5)
+baseline_win = (-1, 0)
+
+
+###############################################################################
+# generate a dics source estimate - see :ref:`ex-inverse-source-power` for
+# more information
+def _gen_dics(active_win, baseline_win, epochs):
+    freqs = np.logspace(np.log10(12), np.log10(30), 9)
+    csd = csd_morlet(epochs, freqs, tmin=-1, tmax=1.5, decim=20)
+    csd_baseline = csd_morlet(epochs, freqs, tmin=baseline_win[0],
+                              tmax=baseline_win[1], decim=20)
+    csd_ers = csd_morlet(epochs, freqs, tmin=active_win[0], tmax=active_win[1],
+                         decim=20)
+    filters = make_dics(epochs.info, fwd, csd.mean(), pick_ori='max-power')
+    baseline_source_power, freqs = apply_dics_csd(csd_baseline.mean(), filters)
+    beta_source_power, freqs = apply_dics_csd(csd_ers.mean(), filters)
+    stc = beta_source_power / baseline_source_power
+    return stc
+
+
+# generate lcmv source estimate
+def _gen_lcmv(active_cov, baseline_cov, common_cov):
+    filters = make_lcmv(epochs.info, fwd, common_cov, reg=0.05,
+                        noise_cov=None, pick_ori='max-power',
+                        weight_norm='nai', rank=None)
+    stc_base = apply_lcmv_cov(baseline_cov, filters)
+    stc_act = apply_lcmv_cov(active_cov, filters)
+    stc_act /= stc_base
+    return stc_act
+
+
+# generate mne/dSPM source estimate
+def _gen_mne(active_cov, baseline_cov, common_cov, fwd, info, method):
+    inverse_operator = make_inverse_operator(info, fwd, common_cov,
+                                             loose=0.2, depth=0.8)
```

Comment:
why `loose=0.2, depth=0.8`? They are the defaults. Just remove them to reduce clutter.